src/components/Admin/UserPage.tsx

```diff
@@ -1,34 +1,43 @@
 import React, { FC } from 'react';
-import { Button, Col, Row } from 'react-bootstrap';
+import { Col, Row } from 'react-bootstrap';
 import { Link, Route, Switch, useRouteMatch } from 'react-router-dom';
 import { useUsersQuery } from '../../generated/graphql';
-import { defaultUser, UserModel } from '../../models/User';
+import { UserModel } from '../../models/User';
+import Button from '../core/Button';
 import Loading from '../core/Loading';
-import User from './User';
-import UserForm, { EditMode } from './UserForm';
+import UserEdit from './UserEdit';
+import { EditMode } from './UserForm';
 import UserList from './UserList';
 
+// type UserPageProps = PageProps;
+
+// export const UserPage: FC<UserPageProps> = ({ paths }) => {
 export const UserPage: FC = () => {
-  const { path } = useRouteMatch();
+  const { path, url } = useRouteMatch();
   const { data, loading } = useUsersQuery();
+
+  // const currentPaths = useMemo(() => [...paths, { value: url, name: 'users', real: true }], [paths]);
+  // useUpdateNavigation({ currentPaths });
+
   const users = (data?.users || []) as UserModel[];
 
   const title = 'Users';
 
   if (loading) return <Loading />;
+
   return (
     <>
       <Row className="justify-content-end">
-        <Col>
-          <h2 style={{ textAlign: 'center' }}>{title}</h2>
-        </Col>
         <Col sm={1}>
           <Link to={`${path}/new`}>
```

Comment:
hide if its already '/new'

Reply:
Good point, but we will fix that with the introduction of the breadcrumbs.

Reply:
I'm adding it to #148